index.html

```diff
@@ -526,9 +537,11 @@
 
 			<div class='stats' id='statsBox'>
 				<div class='stats-title'>
-					Number of Programs in
+					Number of Programs
```

Comment:
The line break in the stats window title is hard-coded here.  If you want to move or remove it, just do that here.